jol-core/src/main/java/org/openjdk/jol/ljv/LJV.java

```diff
@@ -0,0 +1,447 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package org.openjdk.jol.ljv;
+
+//- Author:     John Hamer <J.Hamer@cs.auckland.ac.nz>
+//- Created:    Sat May 10 15:27:48 2003
+
+//- Copyright (C) 2004  John Hamer, University of Auckland
+//-
+//-   This program is free software; you can redistribute it and/or
+//-   modify it under the terms of the GNU General Public License
+//-   as published by the Free Software Foundation; either version 2
+//-   of the License, or (at your option) any later version.
+//-
+//-   This program is distributed in the hope that it will be useful,
+//-   but WITHOUT ANY WARRANTY; without even the implied warranty of
+//-   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//-   GNU General Public License for more details.
+//-
+//-   You should have received a copy of the GNU General Public License along
+//-   with this program; if not, write to the Free Software Foundation, Inc.,
+//-   59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
```

Comment:
This comment is still not addressed.

Reply:
I cannot advise on what you should do here. I see the current PR already does what you suggest, which I presume what John approves? Anyhow, I think we need to ask @robilad for guidance here.

Reply:
Yes, John approves indeed. He's not at GitHub, but he's available via email and he approves the whole idea and he's glad that his idea revived.

Reply:
Idea: can John send the reply to https://mail.openjdk.org/pipermail/jol-dev/2022-September/000464.html then?

Reply:
Thank you for chiming in, @JohnHamer-Glasgow! (And welcome to JOL!)

I have a suggestion to remove `Author` and `Created` lines from this file, because it does not match the style of other files in the project, and this might be lost by accident, or otherwise raise questions during the review. For example, if we ever divide/delete `LJV.java`, it would be hard to judge where these comments should go.

There is a tangential rule that we try to avoid `@author` javadoc tags, to be consistent with OpenJDK code style. I think John is now recorded as developer in project properties, which is cleaner, better, and gives John the credit for the entire project, not just single file.